scripts/mods/Penlight Lua Libraries/pl/utils.lua

```diff
@@ -6,7 +6,7 @@
 -- @module pl.utils
 local format = string.format
 local compat = require 'pl.compat'
-local stdout = io.stdout
+local stdout = function(...) end

```

Comment:
Same as above in array2d.lua:
```suggestion
local stdout = { write = function() end } --io.stdout
```